scss/_dropdown.scss

```diff
@@ -0,0 +1,28 @@
+@import "../node_modules/bootstrap/scss/dropdown";
+
+.layout-heading {
+  position: relative;
+}
+
+#block-contacts-theme-local-actions {
+  top: 0;
+  right: 0;
+  position: absolute;
+  margin-right: map_get($content-padding-parts, x);
+}
+
+.action-group-wrapper,
+.action-group-wrapper li {
+  list-style: none;
+}
+
+.action-group-wrapper button {
+  font-size: 2em;
+  padding: 0.3rem 1.2rem;
+  border-radius: 0 0 0.25rem 0.25rem;
+}
```

Comment:
Can we achieve/look we want with bootstrap's utility classes? e.g. `btn-lg`. Borders could be `rounded-0 rounded-bottom`, or if we have an action group specific CSS, we should use `@include border-top-radius(0);` rather than manually specifying.